knpu/ep3-webpack/production-build.md

```diff
@@ -53,6 +60,20 @@ we will add a new plugin. So, `webpackConfig.plugins.push()` then
 
 [[[ code('cc717660bc') ]]]
 
+***TIP
+This is for Webpack 4. First install `uglifyjs-webpack-plugin` Also, in the great 
```

Comment:
Let me make sure I remember correctly. If we ignore Terser for a moment, in Webpack 4, there are 2 changes:

1) You need to install `uglifyjs-webpack-plugin` - it doesn't come automatically
2) You enable it in a different spot - instead of just adding it to `plugins`, you add it to `optimization.minimizer`.

Is this right? Just making sure.

Also, if terser is the new way of doing it... and using it is simple, maybe we should just show terser and not this plugin at all.